bin/check_sm_processing_job.py

```diff
@@ -0,0 +1,138 @@
+#  Copyright 2024 Amazon.com, Inc. or its affiliates.
```

Comment:
Are we going to migrate all these out of the bin directory?

Reply:
This is a CLI tool to check the progress.